From 4d0b49edad7fb31ebbf03c60a45b72aaa7b7412b Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 7 Jan 2021 14:11:26 +0100
Subject: [PATCH] cmake/OpenJPEGConfig.cmake: replace ABSOLUTE by REALPATH

Fix #1174 by replacing ABSOLUTE by REALPATH which will resolve symlinks:
https://cmake.org/cmake/help/latest/command/get_filename_component.html

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 cmake/OpenJPEGConfig.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git cmake/OpenJPEGConfig.cmake.in cmake/OpenJPEGConfig.cmake.in
index 2925108ad..6e88036c6 100644
--- cmake/OpenJPEGConfig.cmake.in
+++ cmake/OpenJPEGConfig.cmake.in
@@ -32,7 +32,7 @@ if(EXISTS ${SELF_DIR}/OpenJPEGTargets.cmake)
   set(INC_DIR "@CMAKE_INSTALL_PREFIX@/@OPENJPEG_INSTALL_INCLUDE_DIR@")
   file(RELATIVE_PATH PKG_TO_INC_RPATH "${PKG_DIR}" "${INC_DIR}")
 
-  get_filename_component(OPENJPEG_INCLUDE_DIRS "${SELF_DIR}/${PKG_TO_INC_RPATH}" ABSOLUTE)
+  get_filename_component(OPENJPEG_INCLUDE_DIRS "${SELF_DIR}/${PKG_TO_INC_RPATH}" REALPATH)
 
 else()
   if(EXISTS ${SELF_DIR}/OpenJPEGExports.cmake)
